puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_develop.yaml

```diff
@@ -0,0 +1,49 @@
+- job:
+    name: test_develop
+    project-type: matrix
+    logrotate:
+      daysToKeep: -1
+      numToKeep: 32
+    properties:
+      - github_foreman
```

Comment:
It would be used in other jobs such as test_develop_pr_core, test_develop_pr_rubocop but if you think it minor enough it can be inlined in all of them.